yrs/src/transaction.rs

```diff
@@ -578,6 +578,13 @@ impl Transaction {
             }
         }
         // 8. emit 'afterTransactionCleanup'
+        if let Some(transaction_cleanup_events) = store.transaction_cleanup_events.deref() {
+            let event = TransactionCleanupEvent {
+                delete_set: self.delete_set.clone(),
```

Comment:
I suggest that we rename this `after_transaction_event` and share the `delete_set`, `after_state`, and `before_state`. This will be useful for the `UndoManager` implementation.